File: backend/battle/src/main/java/com/paymong/battle/battle/dto/response/BattleMessageResDto.java
package com.paymong.battle.battle.dto.response;

import lombok.Builder;
import lombok.Data;

@Data
@Builder
public class BattleMessageResDto {
    private String battleRoomId;
    private String mongCodeA;
    private String mongCodeB;
    private Integer nowTurn;
    private Integer totalTurn;
    private String nextAttacker;
    private String order;
    private Double damageA;
    private Double damageB;
    private Double healthA;
    private Double healthB;
}

File: android/data/src/main/java/com/paymong/data/dto/response/BattleMessageResDto.kt
package com.paymong.data.dto.response

data class BattleMessageResDto(
    val battleRoomId: String,
    val nowTurn: Int,
    val totalTurn: Int,
    val nextAttacker: String,
    val order: String,
    val damageA: Double,
    val damageB: Double,
    val healthA: Double,
    val healthB: Double
)